CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributor Guidelines:
+
+All contributions to this project should follow the [Nesta Github Guidelines](https://github.com/nestauk/github_support/blob/dev/guidelines/README.md)
+
+## Submitting a Bug Report or a Feature Request
+
+### Reporting a Bug:
+
+Open a new Bug Report in the `nesta_ds_utils` repository.
+
+Include:
+
+- A short description of the bug
+- A small code snippit to reproduce the bug (if possible)
+- If an error or exception is raised include the full traceback
+
+Once you have submitted the issue, post in the [#dev channel](https://nesta.slack.com/app_redirect?channel=dev) with the Issue number so
+people are aware of the bug and can address it asap. If the bug is resolved via Slack (i.e. it is not a bug in the package code)
+please remember to delete the Issue.
+
+### Suggesting a Feature:
+
+First post in the [#dev channel](https://nesta.slack.com/app_redirect?channel=dev) to begin discussion of the feature. A good feature
+for the `nesta_ds_utils` package is:
+
+- General enough that it can be used across projects
+- Does not exist in an existing python package
+
+If the feature is approved, open a new Feature Request in the `nesta_ds_utils` repository. Include:
+
+- A description of the feature
+- A link to an example of the function in an existing project. If an example does not exist, include a detailed description of the
+  desired inputs/outputs/operations.
+
+## Developing a Feature or Addressing a Bug
+
+1. Post in the [#dev channel](https://nesta.slack.com/app_redirect?channel=dev) with the Issue number you plan to address
+2. Clone the repository (we typically use the SSH protocol)
+3. Create a conda environment with python >= 3.8
+4. cd into the top level `nesta_ds_utils` directory
+5. Run `pip install -e ."[dev]"` to install the package with the developer requirements
+6. Run `pre-commit install` to setup pre-commit
+7. Create a new branch corresponding to the issue: `git checkout -b [ISSUE NUMBER]_[BRIEF DESCRIPTION]` (ex: 10_fix_docstrings)
+8. Add new functions to corresponding modules, or new modules if feature doesn't fit within existing modules
+9. Add tests for any new functions to corresponding test module located within tests folder
```

Comment:
Shall we add a bullet point about adding dependencies to `setup.cfg`? Something like: 

- ?. Add any new package requirements in `setup.cfg` under `install_requires`. Requirements which are used only for testing should go under `dev` in `[options.extras_require]`.